docs/package-lock.json

```diff
@@ -24585,9 +24585,9 @@
       }
     },
     "url-parse": {
-      "version": "1.5.10",
-      "resolved": "https://registry.npmjs.org/url-parse/-/url-parse-1.5.10.tgz",
-      "integrity": "sha512-WypcfiRhfeUP9vvF0j6rw0J3hrWrw6iZv3+22h6iRMJ/8z1Tj6XfLP4DsUix5MhMPnXpiHDoKyoZ/bdCkwBCiQ==",
+      "version": "1.5.4",
```

Comment:
This was updated to 1.5.10 recently in [this PR](https://github.com/cosmos/ibc-go/pull/2430). Is it really necessary to downgrade it now?

Reply:
nice catch 🤔 

fwiw I ran `make view-docs` locally and seemed to be fine